### PR TITLE
22 setup home page

### LINKED ISSUE #24: 22-Setup Home Page
- Copy the home page of what Dr. Fuller had
- Don’t implement any login stuff
- Make sure you write unit tests and make sure you get approval before merging
- Estimate: 2 Days

### PR DESCRIPTION
**Pull request by redwing14e**
setup home page

some functionality is missing

### CODE REVIEW

**Review by ralphgregorio — APPROVED** [review]
LGTM!

**Review by cameronbeattie306 — APPROVED** [review]
Looks great! The animation is quite nice. 

I think we'll need to configure the home page in the routes eventually just so they cant see the navbar before they log in but it's fine for now I'm sure

File: frontend/src/HomePage/HomePage.test.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import HomePage from "./HomePage";
+
+test("renders learn react link", () => {
+  const { getByText } = render(<HomePage />);
+  getByText("BEAP ENGINE");
+  getByText("Accurately Process Your Fitness Data");
+  getByText("How To Contribute Data");
+  getByText(
+    "BEAP Engine is a research project developed by Dr. Daniel Fuller",
+    { exact: false },
```
Comment (mpk467) [inline frontend/src/HomePage/HomePage.test.tsx:12]: also works to do getByText((content, element) => content.startsWith('BEAP Engine is a research ')

**Comment by mpk467** [comment]
The animation looks very cool! I see unit tests for rendered components and am happy

**Pull request merged**